docs/usage/self-hosted-experimental.md

```diff
@@ -39,3 +39,10 @@ If set to any string, Renovate will use this as the `user-agent` it sends with H
 
 If set to any value, Renovate will use a "hard" `process.exit()` once all work is done, even if a sub-process is otherwise delaying Node.js from exiting.
 See <https://github.com/renovatebot/renovate/issues/8660> for background on why this was created.
+
+## RENOVATE_X_SERVER_VERSION
+
+If set, renovate will use that string as GitLab server version instead of checking via gitlab api.
+This can be usefull when using the GitlBal `CI_JOB_TOKEN` for running renovate.
```

Comment:
```suggestion
This can be usefull when using the GitLab `CI_JOB_TOKEN` for running renovate.
```

Reply:
I think some expanded explanation here is needed. Have we documented elsewhere how/why we use the GitLab version string?

Reply:
We don't have any docs about version usage yet. Will add to `lib/platform/gitlab/index.md`

*Usages*
- PR `Draft:` prefix instead of `WIP:` (`>= v13.2.0`)
- Truncate markdown to 25K (`< v13.4.0`)
- Add reviewers (`>= v13.9.0`)

Reply:
Another possibility would be an option called `skipPlatformCompatibility` that simply assumes everything is supported. i.e. not requiring someone to put a fixed version in their pipeline which could need changing regularly?

Reply:
It's not ment to be hard-coded, it's to use the `CI_SERVER_VERSION` variable in gitlab pipelines, with will be always defined 😉 

EG: set `RENOVATE_X_SERVER_VERSION: $CI_SERVER_VERSION` in pipeline.